internal/kconfig/kconfig.go

```diff
@@ -39,10 +39,10 @@ func Find() (*os.File, error) {
 }
 
 // Parse parses the kconfig file for which a reader is given.
-// All the CONFIG_* set will be put in the returned map as key with their
-// corresponding value as map value.
+// All the CONFIG_* which are in filteredConfigs and which are set set will be
+// put in the returned map as key with their corresponding value as map value.
```

Comment:
This doesn't document what a `nil` filteredConfigs means. From the tests I gather it means "no filter".